package.json

```diff
@@ -340,8 +340,7 @@
     "@metamask/metamask-eth-abis": "^3.1.1",
     "@metamask/name-controller": "^8.0.0",
     "@metamask/network-controller": "patch:@metamask/network-controller@npm%3A21.0.0#~/.yarn/patches/@metamask-network-controller-npm-21.0.0-559aa8e395.patch",
-    "@metamask/notification-controller": "^6.0.0",
-    "@metamask/notification-services-controller": "^0.11.0",
+    "@metamask/notification-services-controller": "^0.12.0",
```

Comment:
I think this is obvious, but we can't merge this until we solve interface persistence.

Reply:
How come? we aren't implementing detailed view here. That comes in the next PR after we get the interface persistence in.

Reply:
Ah, my bad!